### ISSUE TITLE
Sync of blockchain was fast but sync of lightning-wallet is taking days (2)

### ISSUE THREAD

**Issue by stn021** [raspiblitz/raspiblitz#3197]
Hello,

I think I have a better idea what was wrong. 

In the previous version of this issue I was informed that I have to manually unlock the lnd-wallet.

So far so good. **Except, that I did not set up lnd at all**. I specifically set up raspiblitz to use cln. 
So the lnd-wallet should not be a problem or even be mentioned.
That does sound inconsistent to me. Maybe there is something here to check after all.

And btw: after unlocking lnd the cln-wallet did sync in a reasonable time, even if this does not make sense.
But after then next reboot it started all over at 94%.

This happended with fresh sd-cards on at least two different nodes with different ssd and different sd-cards and different rpi-boards.

**Comment by stn021** [comment]
Also in the cllog there was a line with an error related to the summary-plugin.
I have checked several nodes with cln. This error appears everywhere and every time the summary-plugin is used.

This is not a technical issue because the summary-pluging seems to work without flaw.
I only mention this because it has caused some confusion in a previous thread.

```
2022-06-28T22:02:15.269Z INFO    plugin-summary.py: Plugin summary.py initialized
2022-06-28T22:02:15.461Z UNUSUAL plugin-summary.py: [PriceThread] Cannot allocate write+execute memory for ffi.callback(). You might be running on a system that prevents this. For more information, see https://cffi.readthedocs.io/en/latest/using.html#callbacks
```

**Comment by openoms** [comment]
what is in your `clconf`? 
Is `rescan` present ?

**Comment by stn021** [comment]
> what is in your `clconf`?

I flashed the sd-card, booted and followed the menu. 
Therefore whatever is in `clconf` is the default and was not changed by me.
Any errors I mention appear after the initial setup, purely by the menu and before I have any opportunity to change anything at all.
I can post `clconf` , once you specifically tell me which file or program you mean.

> Is `rescan` present ?

Don't know what that is. I flashed the sd-card, booted and followed the setup-menu. Rescan is not mentioned anywhere.
I can see if rescan is present, once you specifically tell me where to look.
